GameBoard/wwwroot/css/friendsSidebar.css

```diff
@@ -1,15 +1,15 @@
 ﻿:root { --sidebar-width: 200px; }
 
 @media (min-width: 768px) {
-    :root { --sidebar-width: 240px     }
+    :root { --sidebar-width: 240px      }
 }
 
 @media (min-width: 1440px) {
     :root { --sidebar-width: 280px; }
 }
 
 @media (min-width: 2048px) {
-    :root { --sidebar-width: 320px     }
+    :root { --sidebar-width: 320px      }
```

Comment:
What's with the excessive spaces? Also add semicolons like in line 8 while you're at it.

Reply:
I see it's done.